src/components/multi-select.tsx

```diff
@@ -282,6 +282,7 @@ export const MultiSelect = React.forwardRef<
           className="w-auto p-0"
```

Comment:
```suggestion
          className="w-[var(--radix-popper-anchor-width)] p-0"
```
You can also do it in tw 🙂

Reply:
Thanks, I didn't know how to do that with Tw but great to know.
Sorry for duplicate, this can be closed so